libs/rs-sdk/sdk/src/lib.rs

```diff
@@ -21,3 +21,23 @@ pub use secp256k1::secp256k1_verify;
 pub use tally::*;
 
 pub use seda_sdk_macros::oracle_program;
+
+#[cfg(feature = "hide-panic-paths")]
+/// A function run before the main function, via `ctor::ctor`, to set a sanitized panic hook.
+/// This hook takes the `std::panic::PanicInfo` and attempts to downcast the payload to a string.
+/// If unsuccessful, it defaults to printing "<panic>".
+/// It then gracefully aborts the process.
+#[ctor::ctor]
+fn init_sanitized_panic_hook() {
+    std::panic::set_hook(Box::new(|info| {
+        // print only the panic message, never the location
+        let msg = info
+            .payload()
+            .downcast_ref::<&str>()
+            .copied()
+            .or_else(|| info.payload().downcast_ref::<String>().map(|s| s.as_str()))
+            .unwrap_or("<panic>");
+        elog!("{msg}");
```

Comment:
Not sure how feasible it is but can we add a short blurb here? Similar to the RUST_BACKTRACE message? If not no biggy

Reply:
I think I'm not explaining it well 😬 I mean similar to how the default panic tells you how you can get additional details, we could add a short text explaining how to get path info.

Reply:
Ah got ya! I added a short blurb similar to the rust one. Let me know if it looks good!